public/App.js

```diff
@@ -1,6 +1,5 @@
 // Insert your local IP + port here
-// 192.168.1.11
-const socket = new WebSocket('ws://192.168.1.11:8080');
+const socket = new WebSocket('ws://192.168.0.103:7080');
```

Comment:
I would love to finally set up http://homegames.link for this to be automatically discovered somehow, but that's down the road